gamearea.py

```diff
@@ -501,3 +554,27 @@ def start_timeout(self, time, callback=None, force=False):
 
     def redraw(self):
         GObject.idle_add(self.queue_draw)
+    
+    def save_highscore(self):
+        file_path = os.path.join(get_activity_root(), 'data', 'highscore')
+        highscore = [0]
+        if os.path.exists(file_path):
+            with open(file_path, "r") as fp:
+                highscore = fp.readlines()
+
+        int_highscore = int(highscore[0])
+        if not int_highscore > self.score:
+            with open(file_path, "w") as fp:
+                fp.write(str(self.score))
+
+    def load_highscore(self):
+        file_path = os.path.join(get_activity_root(), 'data', 'highscore')
+        if os.path.exists(file_path):
+            with open(file_path, "r") as fp:
+                highscore = fp.readlines()
+            try: 
+                return int(highscore[0])
```

Comment:
Line 561-565, and Line 572-576 could be refactored to another function, to reduce code duplication